package.json

```diff
@@ -10,16 +10,16 @@
     "build": "webpack -p && eslint dist"
   },
   "devDependencies": {
-    "copy-webpack-plugin": "^4.1.1",
-    "eslint": "^4.7.2",
-    "eslint-config-strict": "^14.0.0",
+    "copy-webpack-plugin": "^4.4.1",
+    "eslint": "^4.17.0",
+    "eslint-config-strict": "^14.0.1",
     "eslint-loader": "^1.9.0",
-    "literacy": "^1.1.0",
-    "literacy-loader": "^1.1.0",
+    "literacy": "^1.1.1",
+    "literacy-loader": "^1.1.1",
     "rust-native-wasm-loader": "^0.2.2",
     "wasm-loader": "^1.1.0",
-    "webpack": "^3.6.0",
-    "webpack-dev-server": "^2.9.1"
+    "webpack": "~3.10.0",
```

Comment:
This is the problem. Webpack 3.11 breaks webpack-dev-server use case. Freeze at 3.10 until fixed upstream.